macos/Brewfile

```diff
@@ -63,7 +63,7 @@ brew 'node'
 
 # Tools for work
 # brew 'assume-role'
-brew 'awscli'
+# brew 'awscli'
```

Comment:
The install script seems to always take time updating awscli, so I disabled for now